packages/beacon-node/src/chain/archiveStore/differentialState/plan.ts

```diff
@@ -0,0 +1,34 @@
+import {Slot} from "@lodestar/types";
+import {HierarchicalLayers} from "./hierarchicalLayers.ts";
+
+export type StateRegenPlan = {
+  targetSlot: Slot;
```

Comment:
What's the purpose of `targetSlot`? I don't see it being used anywhere. It can also easily be derived from `diffSlots[-1]` can't it?